src/Rules.Framework/Source/GetRulesArgs.cs

```diff
@@ -1,16 +1,14 @@
 namespace Rules.Framework.Source
 {
     using System;
-    using System.Collections.Generic;
-    using System.Runtime.InteropServices;
-    using System.Text;
 
     internal sealed class GetRulesArgs<TContentType>
     {
+        public bool? Active { get; set; }
```

Comment:
Since we are not filtering on the rules data source, can this property be removed?